src/lib/components/ApplyModal.svelte

```diff
@@ -174,7 +171,7 @@
   }
 
   .duration .number {
-    width: 4rem;
+    width: 5rem;
```

Comment:
Could this be `fit-content`?

Reply:
Potentially, though I thought it'd be nice if it would just be at the right width to fit up to 3 digits in the first place (for 365 days). Makes the layout a bit more stable